### DISCUSSION TITLE
problem error notification

### DISCUSSION THREAD

**Discussion by falfikri** [imamsudarajat04/nongkrongkoding#29] [General]
mam, gimana kalo nama register nya kita pake email sama password dan konfirmasi password aja, nah untuk bagian nama nanti biar usernya edit di bagian user page dia aja nanti, gimana? tapi kalo emang mau tetep pake pas di awal register gapapa juga, tapi emang agak lama aku nyelesain problem yang ngga responsive ini, wkwkwk

**Comment by imamsudarajat04** [comment]
untuk sementara boleh lah input registernya email, password sama konfirmasi password. (Soon) kalau udah jalan mungkin bisa kita ubah lagi

**Reply by falfikri** [reply]
okelah, sekarang pake tiga itu aja dulu, responsive nya tai cok ga bisa di ubah

**Reply by imamsudarajat04** [reply]
sip
